src/server.js

```diff
@@ -1,7 +1,9 @@
+const client = require("./db.js")
 const express = require('express');
 const app = express();
-const port = 3000
+const port = process.env.PORT || 3000
 
 app.listen(port, ()=>{
     console.log(`Team server listening at port ${port}!`)
-})
\ No newline at end of file
+})
+client.connect();
```

Comment:
call the client.connect inside the call back where you listen to the port. incase there are any errors you don't connect. also add a log if the database has connected

Reply:
there was suddenly an error in the node modules and serrver could not run. it says could not find index.js until i rename server.js to index.js and deleted the node modules and package-lock folder and npm install again before the error cleared.